card/mod/standard/set/all/rich_html/menu.rb

```diff
@@ -22,7 +22,8 @@
   end
 
   def menu_path_opts
-    opts = { slot: { home_view: voo.home_view } }
+    opts = { slot: { home_view: (voo.home_view || @slot_view),
```

Comment:
so the difference between `@slot_view` and `home_view` is that if you edit something `@slot_view` becomes `edit` but `home_view` stays unchanged?